Source/plugins/JSONRPC.h

```diff
@@ -34,11 +34,40 @@ namespace PluginHost {
 
         virtual void Activate(IShell* service) = 0;
         virtual void Deactivate() = 0;
-        virtual void Dropped(const uint32_t channelId) = 0;
         virtual void Dropped(const IDispatcher::ICallback* callback) = 0;
     };
 
     class EXTERNAL JSONRPC : public ILocalDispatcher {
+    private:
+        class Notification : public IShell::IJSONRPCLink::INotification {
+        public:
+            Notification(JSONRPC& parent)
+                : _parent(parent)
+            {
+            }
+            ~Notification() = default;
+
+            Notification(const Notification&) = delete;
```

Comment:
As this is new code, we should also delete the move constructor and the move assignment operator.